website/blog/2022-03-04-modifying-blocks.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Modifying blocks - Color Theme
+description: Step-by-step guide how to modify and expand functionality of existing blocks.
+slug: modifying-blocks-color-theme
+authors: iobrado
+date: 2022-03-21
+tags: [eightshift, boilerplate, components, blocks]
+hide_table_of_contents: false
+---
+
+In our previous post, we covered how to use Storybook and WP-CLI for adding additional blocks into your project. This post will walk you step-by-step on how to modify an existing block.
+<!--truncate-->
+
+Since Eightshift Development kit is a starter theme, made for developers to jumpstart and speed up their development, you're welcome to modify files directly in it. There is no need to create a child theme to protect it from updates.
+
+### Modifying a block or a component?
+Since we used Quote block in our previous post, we will continue using it as an example since it's fairly simple. One of the first questions you may ask yourself could be: "Should I modify a component or a block?". And the answer is - it depends.
```

Comment:
```suggestion
Because we used Quote block in our previous post, we will continue using it as an example since it's a fairly simple one. One of the first questions you may ask yourself could be: "Should I modify a component or a block?". And the answer is - it depends.
```